scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1593,22 +1635,34 @@ def verifyDBVersion(version):
 
     global redis_host
     global redis_port
+    global redis_username
     global redis_password
 
     global spin_db_host
+    global spin_db_port
     global spin_db_username
     global spin_db_password
     if spin_db_type == 'redis':     
        redis_host = sys.argv[20]
        redis_port = sys.argv[21]
-       redis_password = sys.argv[22]
+       redis_username = sys.argv[22]
+       redis_password = sys.argv[23]
 
     if spin_db_type == 'sql':
        spin_db_host = sys.argv[20]
-       spin_db_username = sys.argv[21]
-       spin_db_password = sys.argv[22]
-    migrate_data_flag = sys.argv[23]
-    platform_host_url = sys.argv[24]
+       spin_db_port = sys.argv[21]
+       spin_db_username = sys.argv[22]
+       spin_db_password = sys.argv[23]
+    if spin_db_type == 'postgres':
```

Comment:
both of these if blocks (sqland postgres) are doing same assignments so can be clubbed into a single block which will be else part of the spin_db_type == 'redis' block

Reply:
Updated as suggested